### PR TITLE
Jdp210201 26 information service

### PR DESCRIPTION
**Pull request by Bauaser-kun**
Please check your environment variables if application is not building properly.

### CODE REVIEW

File: src/test/resources/application.properties [hunk 1]
```diff
@@ -1,4 +1,12 @@
...
\ No newline at end of file
+spring.datasource.password=sa
+
+# EMAIL CONFIGURATION
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=${MAIL_USERNAME}
+spring.mail.password=${MAIL_PASSWORD}
+spring.mail.properties.mail.smtp.auth=true
+spring.mail.properties.mail.smtp.starttls.enable=true
```
Comment (Ktubbo) [inline src/test/resources/application.properties]: Why you change application.properties in /test/?

Comment (Bauaser-kun) [reply src/test/resources/application.properties]: whilw building terts failed to Load applicationcontext. Only tests.

Comment (Bauaser-kun) [reply src/test/resources/application.properties]: damn those mistypes. Not answering via phone again.

Comment (Bauaser-kun) [reply src/test/resources/application.properties]: I just stacked with the test folder after checking faild builds whithout thinking. I will change the file in main tomorrow. (Or today if Im back early enough)

**Comment by Bauaser-kun** [comment]
I've commented newly added line in OrderController since with our Database we will not have any email adresses to sent message to - so it will always end with NullPointerException

**Comment by Ktubbo** [comment]
> I've commented newly added line in OrderController since with our Database we will not have any email adresses to sent message to - so it will always end with NullPointerException

Why you use everywhere `(Order order, User user)`, while order has own User field?

**Comment by Bauaser-kun** [comment]
> Why you use everywhere `(Order order, User user)`, while order has own User field?

Beacuse it was my fist idea to make interface with methodName(Order order, User user). It didn't struck me at that point that I could take contact Data from Order's 'user' field... Gone with a flow aaaand there you have it. 
Thanks for pointing it out - I will fix it right away.

**Review by Ktubbo — APPROVED** [review]

**Review by Maciek2k3 — APPROVED** [review]

**Pull request merged**